app/helpers/plate_helper.rb

```diff
@@ -60,7 +60,7 @@ def sorted_pre_cap_pool_json(current_plate) # rubocop:todo Metrics/AbcSize
             next unless request.pre_capture_pool
 
             pool_id = request.pre_capture_pool.id
-            pool_store[pool_id] ||= { pool_id: pool_id, order_id: request.order_id, wells: [] }
+            pool_store[pool_id] ||= { pool_id:, order_id: request.order_id, wells: [] }
```

Comment:
also think this truncation when the variable matches the symbol name just looks like the dev has just missed something, I like the more positive reassurance of having the value present!

Reply:
I also read that some IDEs (not sure about VSCode) can render the values inline even though they're not in the actual source code which could help some people adapt.

Reply:
Ok I'll get used to it